src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -170,21 +185,6 @@ const chatReportSelector = (report) =>
         parentReportID: report.parentReportID,
     };
 
-/**
- * @param {Object} [reportActions]
- * @returns {Object|undefined}
- */
-const reportActionsSelector = (reportActions) =>
-    reportActions &&
-    _.map(reportActions, (reportAction) => ({
-        errors: lodashGet(reportAction, 'errors', []),
-        message: [
-            {
-                moderationDecision: {decision: lodashGet(reportAction, 'message[0].moderationDecision.decision')},
-            },
-        ],
-    }));
-
```

Comment:
I think we intentionally used this selector. I am concerned this will cause regression on other parts but I will confirm

Reply:
ok so this was originally introduced in #12083 to speed up performance. Can we try to keep using selector?
This is very important component so we should consider performance regression.

Reply:
Restored.